### PR TITLE
📖 Rename "others..." tab with "Other providers..."

### LINKED ISSUE #9953: Rename Others tabs in the Quick-Start
Had a small conversation here https://kubernetes.slack.com/archives/C8TSNPY4T/p1703707530548349
In the [quickstart guide](https://cluster-api.sigs.k8s.io/user/quick-start#generating-the-cluster-configuration), for cloud providers (at least where I faced it. For me, it was Azure) we need to follow instructions from the `Others...` tab. But the name `Others...` is not much meaningful here. Any thoughts on renaming this?

### PR DESCRIPTION
**Pull request by SD-13**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #9953 
This PR renames the instances of "others..." tab to "Other providers..." in the quickstart guide.

### CODE REVIEW

**Comment by SD-13** [comment]
cc @fabriziopandini @sbueringer

File: docs/book/src/user/quick-start.md [hunk 1]
```diff
@@ -1330,7 +1330,7 @@ clusterctl generate cluster capi-quickstart \
 ```
 
 {{#/tab }}
-{{#tab others...}}
+{{#tab Other providers...}}
```
[the remark this answers is not in the document]
Comment (SD-13) [reply docs/book/src/user/quick-start.md:1333]: I will try to build locally to see if it works

[the remark this answers is not in the document]
Comment (SD-13) [reply docs/book/src/user/quick-start.md:1333]: I built locally and it looks good!

**Comment by SD-13** [comment]
@sbueringer Added your suggested changes!

**Comment by sbueringer** [comment]
Thank you!

Btw. Netlify also hosts a version of this branch 

image


Pages changed > user/quick-start.html - [view](https://deploy-preview-10031--kubernetes-sigs-cluster-api.netlify.app/user/quick-start)

**Comment by SD-13** [comment]
> Thank you!
> 
> Btw. Netlify also hosts a version of this branch
> image
> 
> Pages changed > user/quick-start.html - [view](https://deploy-preview-10031--kubernetes-sigs-cluster-api.netlify.app/user/quick-start)

Ahh, didn't noticed. Thanks for letting me know :)

**Pull request merged**